docs/permissions/permission_use_cases.md

```diff
@@ -155,6 +155,16 @@ that only allows Content items of type **Image**.
 This ensures that only files of type `image` can be uploaded,
 and only to the **Pictures** Folder.
 
+## Taxonomies
+
+You can control what users or user groups can work with taxonomies. If you want users to be able to create and assign taxonomy entry, use the following permissions:
+
+- `assign` - to allow user to tag and untag content
+- `read` -  to see the Taxonomy interface
+- `manage` - to create, edit and delete tags
```

Comment:
```suggestion
- `assign` - allows to tag and untag content
- `read` -  allows to see the Taxonomy interface
- `manage` - allows to create, edit and delete tags
```

Reply:
the form used here to keep consistency with the previous descriptions